addons/mail/static/src/messaging/entity_manager.js

```diff
@@ -0,0 +1,749 @@
+odoo.define('mail.messaging.EntityManager', function (require) {
+'use strict';
+
+const { registry } = require('mail.messaging.entityCore');
+const { many2many, many2one, one2many, one2one } = require('mail.messaging.entityUtils');
+const { patchClassMethods, patchInstanceMethods } = require('mail.messaging.utils');
+
+/**
+ * Object that manage entities, notably their update cycle: whenever some
+ * entities are requested for update (either with static method `create()` or
+ * instance method `update()`), this object process them with direct field &
```

Comment:
Old comment still applicable
```suggestion
 * instance method `update()`), this object processes them with direct field &
```